browser_patches/firefox/BUILD_NUMBER

```diff
@@ -1,2 +1,2 @@
-1298
-Changed: yurys@chromium.org Wed Oct 27 17:58:07 PDT 2021
+1299
```

Comment:
you need to bump the firefox-beta build number as well. Otherwise only firefox will trigger a new build.